common-rendering/src/fixtures/article.ts

```diff
@@ -22,7 +22,6 @@ export const getAllThemes = (format: {
 };
 
 export const getThemeNameAsString = (format: ArticleFormat): string => {
-	return format.theme < 5
-		? ArticlePillar[format.theme]
-		: ArticleSpecial[format.theme];
+	// @ts-expect-error -- ArticleFormat.theme is a union of ArticlePillar and ArticleSpecial, one of these lookups will return a valid name
+	return ArticlePillar[format.theme] ?? ArticleSpecial[format.theme];
```

Comment:
```suggestion
	return ArticlePillar[format.theme] ?? ArticleSpecial[format.theme] ?? "Unkown theme";
```

Reply:
I feel like it makes more sense to use ts-expect-error here, although this scenario should never happen I think it makes more sense to return undefined and have errors rather than return an invalid format?

Reply:
Make sense, especially as this is only used in tests. Should we actually throw an error if `typeof theme === "undefined"`?

Reply:
Yep, that makes sense!

Reply:
```suggestion
	// @ts-expect-error -- ArticleFormat.theme is a union of ArticlePillar and ArticleSpecial, one of these lookups will return a valid name
	const themeName = ArticlePillar[theme] ?? ArticleSpecial[theme];
        if(!themeName) throw new Error("Unknown theme")
        return themeName;
```